pkg/controller/seed/reconciler_test.go

```diff
@@ -111,8 +111,7 @@ var _ = Describe("Reconciler", func() {
 					expectedVPA(namespace),
 				))
 			},
-			Entry("Kubernetes version < 1.26", "1.25.0", false, false, false, false),
-			Entry("Kubernetes version >= 1.26", "1.26.0", true, false, false, false),
+			Entry("Kubernetes version >= 1.27", "1.27.0", true, false, false, false),
```

Comment:
You have to:
- adapt https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/7bc4882adb4585e0ed82397592f58e197cb94c8a/pkg/controller/seed/reconciler.go#L413-L425
- remove the `k8sVersion` at https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/7bc4882adb4585e0ed82397592f58e197cb94c8a/pkg/controller/seed/reconciler.go#L188
- remove https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/086059d37dcd6ad7a5f84e9bd199fff3604bd17e/pkg/controller/seed/reconciler_test.go#L49
- adapt https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/086059d37dcd6ad7a5f84e9bd199fff3604bd17e/pkg/controller/seed/reconciler_test.go#L114-L115